EasyLearn/test/controllers/preferences_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class PreferencesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get preferences_index_url
```

Comment:
I think here is `preferences_path`

Reply:
Oh, we don't need this file, I'm going to delete it.